### ISSUE TITLE
PartRemoval.DROP_ITEMS throws about a missing loot parameter

### ISSUE THREAD

**Issue by Juuxel** [AlexIIL/LibMultiPart#35]
Relevant stacktrace:
```
java.lang.IllegalArgumentException: Missing required parameters: [<parameter libmultipart:additional_parts>]
	at net.minecraft.loot.context.LootContext$Builder.build(LootContext.java:208) ~[minecraft-project-@-mapped.jar:?]
	at alexiil.mc.lib.multipart.impl.PartHolder.createLootContext(PartHolder.java:281) ~[libmultipart-base-0.7.1-pre.6.jar:?]
	at alexiil.mc.lib.multipart.impl.PartHolder.dropItems(PartHolder.java:257) ~[libmultipart-base-0.7.1-pre.6.jar:?]
	at alexiil.mc.lib.multipart.impl.PartHolder.remove(PartHolder.java:234) ~[libmultipart-base-0.7.1-pre.6.jar:?]
```

**Comment by AlexIIL** [comment]
Woops.

**Comment by AlexIIL** [comment]
This should be fixed in 0.7.1-pre.7
